package.json

```diff
@@ -27,9 +27,12 @@
     "generate-icons": "tsx scripts/generate-icons.ts && tsx scripts/generate-og-image.ts",
 
     "mgba:start": "docker compose -f docker/docker-compose.yml up -d --build",
+    "mgba:start:prebuilt": "MGBA_IMAGE=ghcr.io/johndeved/pokemon-save-web/mgba-test-env:latest docker compose -f docker/docker-compose.yml up -d",
     "mgba:run": "docker compose -f docker/docker-compose.yml up --build",
+    "mgba:run:prebuilt": "MGBA_IMAGE=ghcr.io/johndeved/pokemon-save-web/mgba-test-env:latest docker compose -f docker/docker-compose.yml up",
     "mgba:stop": "docker compose -f docker/docker-compose.yml down",
-    "mgba:build": "docker build -f docker/Dockerfile -t mgba-test-env:latest ."
+    "mgba:build": "docker build -f docker/Dockerfile -t mgba-test-env:latest .",
+    "mgba:pull": "docker pull ghcr.io/johndeved/pokemon-save-web/mgba-test-env:latest"
   },
```

Comment:
lets assume we always want to at least try to use the prebuilt image. should this go into compose?